src/Script/Composer/Composer.php

```diff
@@ -42,7 +39,8 @@ public static function removeUnusedServices(
     public static function buildServiceMapping()
     {
         $serviceMapping = [];
-        $source = Aws\manifest();
+        $manifest = __DIR__ . '/../../data/manifest.json.php';
+        $source = include($manifest);
```

Comment:
I think when the `manifest.json.php` file is not found, more should be used `require()`

Reply:
Agree that this should throw an error when the file isn't found.  Thanks!